src/mono/netcore/sample/iOS/Program.csproj

```diff
@@ -17,13 +17,13 @@
   <Target Name="TrickRuntimePackLocation" AfterTargets="ProcessFrameworkReferences">
     <ItemGroup>
       <RuntimePack>
-        <PackageDirectory>$(ArtifactsDir)bin\lib-runtime-packs\$(NetCoreAppCurrent)-iOS-$(Configuration)-$(TargetArchitecture)</PackageDirectory>
+        <PackageDirectory>$(ArtifactBinsDir)microsoft.netcore.app.runtime\$(NetCoreAppCurrent)-iOS-$(Configuration)-$(TargetArchitecture)</PackageDirectory>
```

Comment:
These paths are wrong... you're changing it to just use configuration and not `BuildSettings` anymore:

https://github.com/dotnet/runtime/pull/37600/files#diff-d3bedba39ce6da3e37ca0db83d7ef035R273